### DISCUSSION TITLE
Exception: Unity.IL2CPP.Building.BuilderFailedException:

### DISCUSSION THREAD

**Discussion by zgmtXAX** [pixiv/vroid-sdk-developers#73] [Q&A] [answered]
I build project is failure
error:
Exception: Unity.IL2CPP.Building.BuilderFailedException: C:\Program Files\Unity\Hub\Editor\2020.3.36f1c1\Editor\Data\PlaybackEngines\AndroidPlayer\NDK\toolchains\llvm\prebuilt\windows-x86_64\bin\clang++ @"C:\Users\Administrator\AppData\Local\Temp\tmp8EA4.tmp" -o "D:\UnityProject\framework\GameDayFrameWork_GameFramework\Library\il2cpp_android_arm64-v8a\il2cpp_cache\linkresult_C5038E8E925456BE96C2F2C96931D1A2\libil2cpp.so" -shared -Wl,-soname,libil2cpp.so -Wl,--no-undefined -Wl,-z,noexecstack -Wl,--gc-sections -Wl,--build-id -stdlib=libc++ -static-libstdc++ -target aarch64-linux-android21 -Wl,--wrap,sigaction "C:\Program Files\Unity\Hub\Editor\2020.3.36f1c1\Editor\Data\PlaybackEngines\AndroidPlayer\Variations\il2cpp\Development\StaticLibs\arm64-v8a\baselib.a" -llog -rdynamic -fuse-ld=bfd.exe

Is success when I remove sdk , so sdk support this unity version or NDK?

**Comment by nyamadan** [comment]
VRoid SDK 0.1.4 supports Unity 2020 LTS, but we cannot guarantee that VRoid SDK 0.1.4 will build correctly with your specific version of Unity.

We tested Android builds with Unity 2020.3.45f1 and NDK 19.0.5232133.

Please provide more information about the error or any other relevant details, and we will try our best to assist you further.

**Reply by zgmtXAX** [reply, marked as answer]
I repaired this bug, the reason is NDK memory size is too low,please set Edit-> Preference -> Maxumum JVM heap size Mbytes, and BuidSettting don not select "Script Debugging" is true
